src/simulations/collect.jl

```diff
@@ -65,10 +57,11 @@ By default both keywords are `nothing`, i.e. nothing is collected/aggregated.
   then data are collect if `s ∈ when`. Otherwise data are collected if `when(model, s)`
   returns `true`. By default data are collected in every step.
 * `when_model = when` : same as `when` but for model data.
-* `obtainer = Identity` : with 2 other options, `Copy` & `DeepCopy`. `Copy` is useful for
-when you wish to save a container (e.g. an `Array`) to the DataFrame, or a `struct` that
-you have implemented [Base.copy](https://docs.julialang.org/en/v1/base/base/#Base.copy) for.
-`DeepCopy` is for when you need to save a
+* `obtainer = identity` : the method used to transfer information from the `model` to
```

Comment:
I'd suggest:

"to transfer collected data to the `DataFrame`. Typically only change this to `copy` if some data are mutable containers (e.g. `Vector`) which change during evolution, or `deepcopy` if some data are nested mutable containers."

(but keep all hyperlinks, they rock!)